lib/Controller/SettingsController.php

```diff
@@ -172,6 +174,28 @@ private function setShowTasks(string $value):JSONResponse {
 		return new JSONResponse();
 	}
 
+	/**
+	 * Set config for attachments folder
+	 *
+	 * @param string $value
+	 * @return JSONResponse
+	 */
+	private function setAttachmentsFolder(string $value):JSONResponse {
+
+		try {
+			$this->config->setUserValue(
+				$this->userId,
+				$this->appName,
```

Comment:
Once the attachment part will be handled on the server by [Managed Attachments](https://datatracker.ietf.org/doc/html/rfc8607), it would be nice to keep this setting value, so we could set the preference in `dav` instead of `calendar` (`$this->appName`).

Reply:
Simply change to 

```
$this->config->setUserValue(
				$this->userId,
				'dav',
				'attachmentsFolder',
				$value
			);
			```

?

Reply:
https://github.com/nextcloud/calendar/pull/4251/commits/5a5638c6ff7a1b7d336b1fc8a90b913a8d32bb8b